### PR TITLE
Update modern-auth-for-office-2013-and-2016.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Office 2019 to the document as there is no specific doc for 2019 which talks about modern auth

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/issues/880

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/modern-auth-for-office-2013-and-2016.md [hunk 1]
```diff
@@ -25,11 +25,11 @@ ms.collection:
 description: "Learn how Office 365 modern authentication works differently for Office 2013 and 2016 client apps."
 ---
 
-# How modern authentication works for Office 2013 and Office 2016 client apps
+# How modern authentication works for Office 2013, Office 2016 and Office 2019 client apps
```
Comment (JohanFreelancer9) [inline Enterprise/modern-auth-for-office-2013-and-2016.md:28]:
```suggestion
# How modern authentication works for Office 2013, Office 2016, and Office 2019 client apps
```

File: Enterprise/modern-auth-for-office-2013-and-2016.md [hunk 2]
```diff
@@ -25,11 +25,11 @@ ms.collection:
...
 
 *This article applies to both Office 365 Enterprise and Microsoft 365 Enterprise.*
 
-Read this article to learn how Office 2013 and Office 2016 client apps use modern authentication features based on the authentication configuration on the Office 365 tenant for Exchange Online, SharePoint Online, and Skype for Business Online.
+Read this article to learn how Office 2013, Office 2016 and Office 2019 client apps use modern authentication features based on the authentication configuration on the Office 365 tenant for Exchange Online, SharePoint Online, and Skype for Business Online.
```
Comment (JohanFreelancer9) [inline Enterprise/modern-auth-for-office-2013-and-2016.md:32]:
```suggestion
Read this article to learn how Office 2013, Office 2016, and Office 2019 client apps use modern authentication features based on the authentication configuration on the Office 365 tenant for Exchange Online, SharePoint Online, and Skype for Business Online.
```

File: Enterprise/modern-auth-for-office-2013-and-2016.md [hunk 3]
```diff
@@ -77,6 +77,10 @@ The following table describes the authentication behavior for Office 2013 or Off
   
 |****Office client app version****|****Registry key present?****|****Modern authentication on?****|****Authentication behavior with modern authentication turned on for the tenant (default)****|****Authentication behavior with modern authentication turned off for the tenant****|
 |:-----|:-----|:-----|:-----|:-----|
+|Office 2019  <br/> |No, <br> AlwaysUseMSOAuthForAutoDiscover = 1 <br/> |Yes  <br/> |Forces modern authentication on Outlook 2010, 2013 or 2016 or 2019 <br/> [More info](https://support.microsoft.com/help/3126599/outlook-prompts-for-password-when-modern-authentication-is-enabled)|Forces modern authentication within the Outlook client.<br/> |
```
Comment (JohanFreelancer9) [inline Enterprise/modern-auth-for-office-2013-and-2016.md:80]:
```suggestion
|Office 2019  <br/> |No, <br> AlwaysUseMSOAuthForAutoDiscover = 1 <br/> |Yes  <br/> |Forces modern authentication on Outlook 2010, 2013, or 2019 <br/> [More info](https://support.microsoft.com/help/3126599/outlook-prompts-for-password-when-modern-authentication-is-enabled)|Forces modern authentication within the Outlook client.<br/> |
```

File: Enterprise/modern-auth-for-office-2013-and-2016.md [hunk 4]
```diff
@@ -77,6 +77,10 @@ The following table describes the authentication behavior for Office 2013 or Off
...
+|Office 2019  <br/> |Yes, EnableADAL = 1  <br/> |Yes  <br/> |Modern authentication is attempted first. If the server refuses a modern authentication connection, then basic authentication is used. Server refuses modern authentication when the tenant is not enabled.  <br/> |Modern authentication is attempted first. If the server refuses a modern authentication connection, then basic authentication is used. Server refuses modern authentication when the tenant is not enabled.  <br/> |
+|Office 2019  <br/> |Yes, EnableADAL=0  <br/> |No  <br/> |Basic authentication  <br/> |Basic authentication  <br/> |
 |Office 2016  <br/> |No, <br> AlwaysUseMSOAuthForAutoDiscover = 1 <br/> |Yes  <br/> |Forces modern authentication on Outlook 2010, 2013 or 2016 <br/> [More info](https://support.microsoft.com/help/3126599/outlook-prompts-for-password-when-modern-authentication-is-enabled)|Forces modern authentication within the Outlook client.<br/> |
```
Comment (JohanFreelancer9) [inline Enterprise/modern-auth-for-office-2013-and-2016.md:84]:
```suggestion
|Office 2016  <br/> |No, <br> AlwaysUseMSOAuthForAutoDiscover = 1 <br/> |Yes  <br/> |Forces modern authentication on Outlook 2010, 2013, or 2016 <br/> [More info](https://support.microsoft.com/help/3126599/outlook-prompts-for-password-when-modern-authentication-is-enabled)|Forces modern authentication within the Outlook client.<br/> |
```

**Comment by mypil** [comment]
@MSFTTracyP - PR has been copy edited and is ready for final review. This is based on issue #880.

Thank you.

**Comment by yogkumgit** [comment]
@msbemba  Thank you very much for the contribution and updating the documentation with PR. Appreciate.

**Pull request merged**